tests/TestHarness/Cluster.py

```diff
@@ -226,9 +226,12 @@ def launch(self, pnodes=1, unstartedNodes=0, totalNodes=1, prodCount=1, topo="me
         if sharedProducers > 0:
             producerFlag += (" --shared-producers %d" % (sharedProducers))
 
+        if maximumClients <= 0:
+            maximumClients = 25
         if maximumP2pPerHost <= 0:
             maximumP2pPerHost = totalNodes
```

Comment:
Should the default for `maximumP2pPerHost` be `totalNodes` in the parameter list since `0` will be overwritten anyway?

Reply:
I don't think I am able to set one argument equal to another as a default in the declaration

Reply:
Great point, I didn't think that through completely.  Maybe just say in the doc string that it defaults to the `totalNodes` value if left at `0`